src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        return new Ball(colorSupplier.getRandomColor(), new Random().nextInt(100));
```

Comment:
```suggestion
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(100));
```

Мы создавали поля random и colorSupplier что б использовать их в методе getRandomBall() и не создавать каждый раз новый объект, при вызове этого метода